apps/src/Register2D.cpp

```diff
@@ -91,16 +99,40 @@ int main(int argc, char* argv[])
     // Setup final transform
     auto compositeTrans = CompositeTransform::New();
     ///// Landmark Registration /////
-    if (parsed.count("landmarks")) {
-        printf("Running affine registration...\n");
-        fs::path landmarksFileName = parsed["landmarks"].as<std::string>();
-        LandmarkReader landmarkReader(landmarksFileName);
-        landmarkReader.setFixedImage(fixedImage);
-        landmarkReader.setMovingImage(movingImage);
-        landmarkReader.read();
-        auto fixedLandmarks = landmarkReader.getFixedLandmarks();
-        auto movingLandmarks = landmarkReader.getMovingLandmarks();
+    if (parsed.count("disable-landmark") == 0) {
+        LandmarkContainer fixedLandmarks, movingLandmarks;
+        if (parsed.count("input-landmarks") > 0) {
```

Comment:
```suggestion
        if (parsed.count("input-landmarks")) {
```

Reply:
Does this do the same thing?

Reply:
It does. I got in the habit of avoiding it because implicit conversion from int to bool might mask a bug.